packages/react-components/react-combobox/src/stories/Dropdown/DropdownAccessibility.md

```diff
@@ -0,0 +1,21 @@
+<details>
+<summary>
+ Accessibility
+</summary>
+
+### Do
+
+- **Provide a label for the Dropdown.** This can be done either by using the `DropdownField` component with a `label` prop, or by using a [custom labelling technique](#TODO).
+- **Consider using Select or a Checkbox group.** For simple single-select use cases, consider using `Select` for better accessibility and mobile support. For simple multiselect use cases with less than 10 options, consider using a group of `Checkbox` components.
```

Comment:
maybe flip this and say "Do use Dropdown when you need JSX options, otherwise use Select instead. Then another saying "Do use Multi select dropdown when you have 10+ options, otherwise consider using a Checkbox group instead".

Reply:
I like that better, thanks!